LICENSE

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Is this blank line necessry?

Reply:
Matches the license we use for the driver